python/ctsm/crop_calendars/cropcal_utils.py

```diff
@@ -9,6 +9,25 @@
 from ctsm.utils import is_instantaneous
 
 
+def log(logger_in, string):
```

Comment:
This doesn't need to be done now, but I wonder if these shouldn't be general use methods that are added to the ctsm_logging.py module?

Reply:
Actually, similarly with the leading_datetime_string function above -- should it be moved to one of the general ctsm utils?

Reply:
Done with ed3223fca.